src/server.c

```diff
@@ -173,7 +173,7 @@ void _serverLog(int level, const char *fmt, ...) {
  * We actually use this only for signals that are not fatal from the point
  * of view of Redis. Signals that are going to kill the server anyway and
  * where we need printf-alike features are served by serverLog(). */
-void serverLogFromHandler(int level, const char *msg) {
+static void _serverLogFromHandler(int level, const char *msg) {
```

Comment:
to avoid copying the message if there is no format?
my intention was not to change all the existing calls to `serverLogFromHandler`

Reply:
and completely remove the one that takes `fmt, ...` and uses `_safe_vsnprintf`?
that may be ok, but i think we may want to use it some day, so having the two APIs in both modes, and naming them consistently seems better.

Reply:
The original version of `serverLogFromHandler` did not take `fmt, ...`, the varidaic args were added by me.
What I'm saying is that i didn't want to replace all the existing calls with `_serverLogFromHandler` (or whatever new name we'll choose), and since `...` also works for passing only a string literal, i moved the implementation to the static `_serverLogFromHandler`, and used  `serverLogFromHandler` as a wrapper.